pandunia-loge.csv

```diff
@@ -1567,6 +1568,7 @@ judau||||judo||yudo|||||||||||||ĵudo|judo|judo
 jude||||judgment (sentence)||fallo (juicio, criterio)|||||||||||||juĝo|tuomio|wyrok
 juder|||eng:judge, fra:juge, spa:juez, por:juiz, hin:जज (jaj), ben:জজ (jôj), swa:jaji + rus:судья (sud'ya)|judge (referee)|juje|juez (árbitro, réferi)|juiz|судья|||||||||||juĝisto|tuomari|sędzia
 judi||||judicial||||||||||||||||oikeudellinen|
+juyi|||zho:主义 (zhǔyì), yue:主義 (zyu2 ji6), wuu:主義 (tsr nyi2), jpn:主義 (shugi), kor:주의 (juui), vie:chủ nghĩa|ideology (principle, doctrine, -ism)||ideología (doctrina, -ismo)||идеология|||主义|主義|||||||ismo|oikeudellinen (aate, ismi, ideologia, uskonto, jne)|ideologii
```

Comment:
In my opinion this word should be only one syllable. Also we don't really need to allow syllables like "yi" anymore.

Reply:
that make sense.  what about **jui**, then?

Reply:
That sounds better and I don't have any better ideas myself. So OK. :)